specification/conformance/conformance_rules/columns/tags.json

```
@@ -0,0 +1,427 @@
+{
+    "Tags-C-000-C": {
+        "Function": "Composite",
+        "Reference": "Tags",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [
+            "TAGGING_SUPPORTED"
+        ],
+        "Type": "Static",
+        "ValidationCriteria": {
+            "MustSatisfy": "The Tags column adheres to the following requirements:",
+            "Keyword": "MUST",
+            "Requirement": {
+                "CheckFunction": "AND",
+                "Items": [
+                    {
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "Tags-C-001-M"
+                    },
+                    {
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "Tags-C-002-O"
+                    },
+                    {
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "Tags-C-003-C"
+                    },
+                    {
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "Tags-C-011-M"
+                    },
+                    {
+                        "CheckFunction": "CheckConformanceRule",
+                        "ConformanceRuleId": "Tags-C-014-M"
+                    }
+                ]
+            },
+            "Condition": {},
+            "Dependencies": [
+                "CostAndUsage-D-064-C",
+                "Tags-C-001-M",
+                "Tags-C-002-O",
+                "Tags-C-003-C",
+                "Tags-C-011-M",
+                "Tags-C-014-M"
+            ]
+        }
+    },
+    "Tags-C-001-M": {
+        "Function": "Type",
+        "Reference": "Tags",
+        "EntityType": "Column",
+        "Notes": "",
+        "CRVersionIntroduced": "1.2",
+        "Status": "Active",
+        "ApplicabilityCriteria": [],
+        "Type": "Static",
+        "ValidationCriteria": {
+            "MustSatisfy": "Tags MUST be of type KeyValueFormat.",
+            "Keyword": "MUST",
+            "Requirement": {
+                "CheckFunction": "FormatKeyValue",
+                "ColumnName": "Tags"
+            },
+            "Condition": {},
+            "Dependencies": []
+        }
+    },
```

similar to [this](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/1209/files#r2412947365)

```suggestion
    "Tags-C-001-M": {
        "Function": "Format",
        "Reference": "Tags",
        "EntityType": "Column",
        "Notes": "",
        "CRVersionIntroduced": "1.2",
        "Status": "Active",
        "ApplicabilityCriteria": [],
        "Type": "Static",
        "ValidationCriteria": {
            "MustSatisfy": "Tags MUST conform to KeyValueFormat requirements.",
            "Keyword": "MUST",
            "Requirement": {
                "CheckFunction": "FormatKeyValue",
                "ColumnName": "Tags"
            },
            "Condition": {},
            "Dependencies": []
        }
    },
```